From c5a6039bbbdc5da44a0eed6775c924e667d94f54 Mon Sep 17 00:00:00 2001
From: JoyingKyc <joying.kuo@mediatek.com>
Date: Thu, 28 Apr 2022 17:56:55 +0800
Subject: [PATCH] tensorflow-lite: add external delegate

Add External Delegate for building libtensorflowlite.so to support the demo APP (Gst-inference)
---
 tensorflow/lite/BUILD | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/tensorflow/lite/BUILD b/tensorflow/lite/BUILD
index 72a46e6675f..d3c6e7ef008 100644
--- a/tensorflow/lite/BUILD
+++ b/tensorflow/lite/BUILD
@@ -309,6 +309,7 @@ cc_library(
         ":string",
         ":type_to_tflitetype",
         ":util",
+        "//tensorflow/lite/delegates/external:external_delegate",
         "//tensorflow/lite/c:common",
         "//tensorflow/lite/core/api",
         "//tensorflow/lite/core/api:verifier",
@@ -348,6 +349,7 @@ cc_library(
         ":string",
         ":type_to_tflitetype",
         ":util",
+        "//tensorflow/lite/delegates/external:external_delegate",
         "//tensorflow/lite/c:common",
         "//tensorflow/lite/core/api",
         "//tensorflow/lite/core/api:verifier",
@@ -414,6 +416,7 @@ cc_library(
         ":type_to_tflitetype",
         ":util",
         ":version",
+        "//tensorflow/lite/delegates/external:external_delegate",
         "//tensorflow/lite/c:c_api_types",
         "//tensorflow/lite/c:common",
         "//tensorflow/lite/core/api",
@@ -473,6 +476,7 @@ cc_library(
         ":string",
         ":type_to_tflitetype",
         ":util",
+        "//tensorflow/lite/delegates/external:external_delegate",
         "//tensorflow/lite/c:c_api_types",
         "//tensorflow/lite/c:common",
         "//tensorflow/lite/core/api",
-- 
2.18.0

